src/composables/info/feature-info.composable.ts

```diff
@@ -0,0 +1,413 @@
+          isLongPress.value = false
+          return
+        }
+
+        if (
+          // TODO: integrate in v3 and migrate once available
+          // routingOpen ||
+          // lidarOpen ||
+          // appActivetool.value.isActive() || => corresponds to: measureActive, streetviewActive
+          drawStateActive.value ||
+          editStateActive.value ||
+          isLoading.value
+        ) {
+          return
+        }
+
+        timeoutId.value = window.setTimeout(() => {
+          let found = false
+          if (
+            (layersOpen.value || myMapsOpen.value) &&
+            drawnFeatures.value.length > 0
+          ) {
+            found = checkForDrawnFeature(evt.pixel)
+          }
+
+          if (!found && startPixel.value && stopPixel.value) {
+            featureInfoLayerService.clearFeatures()
+            const deltaX = Math.abs(startPixel.value[0] - stopPixel.value[0])
+            const deltaY = Math.abs(startPixel.value[1] - stopPixel.value[1])
+            if (deltaX + deltaY < 6) {
+              singleclickEvent(evt)
+              startPixel.value = null
+              stopPixel.value = null
+            }
+          }
+        }, 500)
+      })()
+    })
+
+    listen(map, 'pointermove', event => {
+      const evt = event as MapBrowserEvent<any>
+      if (evt.dragging || isLoading.value) {
+        return
+      }
+      if (
+        drawStateActive.value /*TODO: || measureActive || streetviewActive*/
+      ) {
+        map.getViewport().style.cursor = ''
+      } else {
+        const pixel = map.getEventPixel(evt.originalEvent)
+        let hit: boolean | undefined = false
+        try {
+          hit = map.forEachLayerAtPixel(
+            pixel,
+            layer => {
+              if (layer) {
+                const metadata = layer.get('metadata')
+                if (
+                  metadata &&
+                  metadata['is_queryable'] &&
+                  layer.getVisible() &&
+                  layer.getOpacity() > 0
+                ) {
+                  return true
+                }
+              }
+              return false
+            },
+            {
+              layerFilter: layer => !!layer.getSource(),
+            }
+          )
+        } catch (error) {
+          hit = false
+        }
+        map.getViewport().style.cursor = hit ? 'pointer' : ''
+      }
+    })
+
+    watchEffect(() => {
+      if (fid.value) {
+        ;(async () => {
+          await getFeatureInfoById(fid.value as string)
+        })()
+      }
+    })
+  }
+
+  async function getFeatureInfoById(fid: string): Promise<void> {
+    const fids = fid.split(',')
+    for (const curFid of fids) {
+      const splittedFid = curFid.split('_')
+      const layersList = [splittedFid[0]]
+      const layerLabel: { [key: string]: string } = {}
+
+      try {
+        if (layersList.length > 0) {
+          setLoading(true)
+          map.getViewport().style.cursor = 'wait'
```

Comment:
is the style cursor going back to default point in this function? I don't see where. Maybe the cursor style can be handled in a watch on loading ref.

Reply:
Yep, this happens in `reset()` or now called `done()`